src/MemberProfile.js

```diff
@@ -1,24 +1,129 @@
 import React from "react";
-import { Link } from "react-router-dom";
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import Chip from 'material-ui/Chip';
 
-const MemberProfile = ({ user }) => {
+const MemberProfile = ({user}) => {
   if (!user) {
     return (
       <div>
         Loading...
       </div>
     )
   }
+  const {picture, name, description, githubName, flowdockName, email} = user;
   return (
-    <div>
-      <img src={user.picture} alt="avatar" />
-      <h1>{user.name}</h1>
-      <p>{user.description}</p>
-      <p>{user.email}</p>
-      <p>GitHub: {user.githubName}</p>
-      <p>Flowdock: {user.flowdockName}</p>
-      <Link to="/me/edit">edit</Link>
-    </div>
+    <MuiThemeProvider>
```

Comment:
This `<MuiThemeProvider>` component should probably live in the `App` component so we can take advantage these components anywhere throughout the app.